canvas_modules/common-canvas/src/object-model/layout-dimensions.js

```diff
@@ -73,7 +73,7 @@ const portsHorizontalDefaultLayout = {
 		labelAllowReturnKey: false, // true allows line feed to be inserted into label, "save" to make the return key save the label.
 
 		// An array of decorations to be applied to the node. For details see:
-		// https://github.com/elyra-ai/canvas/wiki/2.4.2-Decoration-Specification
+		// https://github.com/elyra-ai/canvas/tree/main/guide/2.4.2-Decoration-Specification.md
```

Comment:
Shouldn't all these links be github pages links?  Also, I don't this `guide` is the folder anymore.

Reply:
Hi Matt, I have reverted the docs links to point to existing elyra canvas docs once the new one is deployed I will change these urls pointing to new docs.